lib/paperclip-globalize3.rb

```diff
@@ -4,12 +4,25 @@
 require "globalize"
 require "paperclip"
 
+# Paperclip locale interpolation: if locale fallbacks are used, we need to determine & use the fallback locale
 Paperclip.interpolates(:locale) { |attachment, _|
-  if attachment.instance.send("#{attachment.name}_file_name").respond_to?(:translation_metadata)
-    attachment.instance.send("#{attachment.name}_file_name").translation_metadata[:locale].to_s
+  record = attachment.instance
+  file_name_attr = "#{attachment.name}_file_name"
+  attachment_locale =
+  if record.respond_to?(:translation) && record.translated?(file_name_attr)
```

Comment:
Could you please review indentation here? (+ maybe improve code style by using do/end for the whole interpolates block...) Thx!